src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -0,0 +1,295 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  EventEmitter,
+  forwardRef,
+  Input,
+  OnDestroy,
+  OnInit,
+  Output,
+  ViewChild
+} from '@angular/core';
+
+import {
+  AbstractControl,
+  ControlValueAccessor,
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  NG_VALUE_ACCESSOR,
+  ValidationErrors
+} from '@angular/forms';
+
+import {
+  SkyAppWindowRef
+} from '@skyux/core';
+
+import 'intl-tel-input';
+
+import {
+  fromEvent,
+  Subject
+} from 'rxjs';
+
+import {
+  SkyAutocompleteInputDirective,
+  SkyAutocompleteSelectionChange
+} from '../autocomplete';
+
+import {
+  SkyCountryFieldCountry
+} from './types';
+
+@Component({
+  selector: 'sky-country-field',
+  templateUrl: './country-field.component.html',
+  styleUrls: ['./country-field.component.scss'],
+  providers: [{
+    provide: NG_VALUE_ACCESSOR,
+    /* tslint:disable-next-line:no-forward-ref */
+    useExisting: forwardRef(() => SkyCountryFieldComponent),
+    multi: true
+  }],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy, OnInit {
+
+  /**
+   * Specifies the [International Organization for Standardization Alpha 2](https://www.nationsonline.org/oneworld/country_code_list.htm)
+   * country code for the default country.
+   * This default country will show up at the top of search results when applicable.
```

Comment:
I'd change "This" to "The" at the start. And I'd change "will show" to "appears" to use active voice.

More importantly, what do we mean by "when applicable"? This seems vague to me. I *assume* this means when the default country is included in search results, but I'm not sure. Whatever it means, I'd update this description to be specific. If my guess about what we mean is correct, I'd change this to `When search results include the default country, it appears at the top of the list.`

Reply:
I made this change.